src/actionTypes/notificationActionStyles.js

```diff
@@ -0,0 +1,3 @@
+export const CREATE_ONE = 'CREATE_ONE';
+
+export const CATCH_ERROR = 'CATCH_ERROR';
```

Comment:
I suggest you rename this file `notificationActionStyles.js` according to what it does. It is not **styles**

Reply:
Good idea.. thank you